intro-to-git/git-cheatsheet.md

```diff
@@ -1 +1,17 @@
-# Git Cheatsheet
\ No newline at end of file
+# Git Cheatsheet
+
+[ Comprehensive Git Commands Cheat Sheet](https://www.atlassian.com/git/tutorials/atlassian-git-cheatsheet)
+
+
+| Command | Description |
+|--|--|
+| `git clone <repo>` | Clone repo located at `<repo>` onto local machine. Original repo can be located on the local filesystem or on a remote machine via HTTP or SSH. |`git clone https://github.com/AdaGold/tdd-exercise.git`|
+|`git status`| List which files are staged, unstaged, and untracked |
+|`git add <directory>`| Stage all changes in `<directory>` for the next commit. Replace `<directory>` with a `<file>` to change a specific file. |
```

Comment:
Maybe a note here that '.' is a shorthand for the current directory?

Reply:
Added a line item for this.